xchange-lgo/pom.xml

```diff
@@ -0,0 +1,46 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.knowm.xchange</groupId>
+        <artifactId>xchange-parent</artifactId>
+        <version>4.3.21-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>xchange-lgo</artifactId>
+
+    <name>XChange LGO</name>
+    <description>XChange implementation for LGO</description>
+
+    <url>http://knowm.org/open-source/xchange/</url>
+    <inceptionYear>2019</inceptionYear>
+
+    <organization>
+        <name>Knowm Inc.</name>
+        <url>http://knowm.org/open-source/xchange/</url>
+    </organization>
+
+    <!-- Parent provides default configuration for dependencies -->
+    <dependencies>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
seams not to be used in this module and therefore can be removed

Reply:
Used in LgoSignatureServiceLocalRsa.java